Can't build with rust on x86-64

Using rustc 1.46.0-nightly (346aec9b0 2020-07-11). When attempting to build Quiche as a dependency:
```
error: failed to run custom build command for `quiche v0.5.1`

Caused by:
  process didn't exit successfully: `C:\Users\ethin\source\moo\target\debug\build\quiche-8fa76ddc5467a99f\build-script-build` (exit code: 101)
  --- stdout
  running: "cmake" "C:\\Users\\ethin\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\quiche-0.5.1\\deps/boringssl" "-G" "Visual Studio 16 2019" "-Thost=x64" "-Ax64" "-DCMAKE_INSTALL_PREFIX=C:\\Users\\ethin\\source\\moo\\target\\debug\\build\\quiche-fa2385dcf3f53172\\out" "-DCMAKE_C_FLAGS= -nologo -MD -Brepro" "-DCMAKE_C_FLAGS_DEBUG= -nologo -MD -Brepro" "-DCMAKE_CXX_FLAGS= -nologo -MD -Brepro" "-DCMAKE_CXX_FLAGS_DEBUG= -nologo -MD -Brepro" "-DCMAKE_ASM_FLAGS= -nologo -MD -Brepro" "-DCMAKE_ASM_FLAGS_DEBUG= -nologo -MD -Brepro" "-DCMAKE_BUILD_TYPE=Debug"
  -- Configuring done

  --- stderr
  CMake Error at CMakeLists.txt:344 (add_library):
    Cannot find source file:

      win-x86_64/crypto/chacha/chacha-x86_64.asm

    Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
    .hpp .hxx .in .txx


  CMake Error at CMakeLists.txt:344 (add_library):
    No SOURCES given to target: crypto


  CMake Generate step failed.  Build files cannot be regenerated correctly.
  thread 'main' panicked at '
  command did not execute successfully, got: exit code: 1

  build script failed, must exit now', C:\Users\ethin\.cargo\registry\src\github.com-1ecc6299db9ec823\cmake-0.1.44\src\lib.rs:885:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
```
Not sure how to go about fixing this.
Edit: OK, so apparently there's code for iOS (aarch64 and ARM), Linux (Aarch64, ARM, PPC64, x86, and x86_64), and Mac (x86 and x86_64) but not for windows at all.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,6 +17,7 @@ include = [
     "/COPYING",
     "/benches",
     "/deps/boringssl/**/*.[chS]",
+    "/deps/boringssl/**/*.asm",
     "/deps/boringssl/src/**/*.cc",
     "/deps/boringssl/**/CMakeLists.txt",
     "/deps/boringssl/**/sources.cmake",

